### DISCUSSION TITLE
PrimeReact Tree: Missing features

### DISCUSSION THREAD

**Discussion by oravecz** [primefaces/community#157] [PrimeReact] [answered]
Is it correct to say that the PrimeReact Tree is currently missing the following capabilities

1. Virtual scrolling/rendering for large lists
2. Drag/drop between trees

**Comment by habubey** [comment, marked as answer]
I've opened a [ticket](https://github.com/primefaces/primereact/issues/5004) for this new feature request. Thanks for the suggestion.
